tests/numpy/test_numpy_wrapper.py

```diff
@@ -428,6 +431,7 @@ def __call__(self, *args, **kwargs):
         assert len(res) == 2
 
 
+@pytest.mark.integration_test

```

Comment:
I'm no expert on this, but this could be a system test just as well, no? I know that best practices are not the subject of this PR, but maybe there is another example that is more clearly an integration and not a system test? (not sure we have those :sweat_smile:

Reply:
Yep, best practices guidelines are still being defined ---- it's a tough boundary to cross, and we tend to call system tests unit tests throughout. The goal will be to define what best works for us. My idea was anything that touches within a module to be integration, and anything that crosses multiple modules to be system. But, we can better define that in a follow-up meeting. I'll be bringing this up for discussion as the next group meeting.

Reply:
Sounds good, resolving :)